File: MODUL 2/PRAK204-210817210005-Muhammad-Irgi-Fahrezha.c
#include <stdio.h>


int main () {

    float r;
    float t;
    float volume;
    float luas;
    float keliling;
    float pi = 22.0/7.0;

    scanf("%f", &r);
    scanf("%f", &t);

    volume = pi * r * r * t;
    luas = 2 * pi * r *(r+t);
    keliling = 2 * pi * r;

    printf("Volume = %.2f\n", volume);
    printf("Luas = %.2f\n", luas);
    printf("Keliling = %.2f\n", keliling);
}

File: MODUL 2/PRAK204-210817210005-Muhammad-Irgi-Fahrezha.py
pi = float(22/7)
angka = []
while True:
    line = input()
    if not line:
        break
    angka.extend(line.split())

angka = list(map(float, angka))

r, t = angka

volume = pi * r * r * t
luas = 2 * pi * r * (r + t)
keliling = 2 * pi * r

print (f"Volume = {volume:.2f}")
print (f"Luas = {luas:.2f}")
print (f"Keliling = {keliling:.2f}")